lib/Fmt_ast.ml

```diff
@@ -3103,15 +3109,16 @@ and fmt_tydcl_params c ctx params =
     ( wrap_fits_breaks_if ~space:false c.conf
         (List.length params > 1)
         "(" ")"
-        (list params (Params.comma_sep c.conf) (fun (ty, vc) ->
-             fmt_variance vc $ fmt_core_type c (sub_typ ~ctx ty) ) )
+        (list params (Params.comma_sep c.conf) (fun (ty, (vc, inj)) ->
+             fmt_variance vc $ fmt_injectivity inj
```

Comment:
it's minor but since variance and injectivity are always formatted together, what do you think about passing the tuple to a `fmt_variance_and_injectivity` function?

Reply:
Done